### ISSUE TITLE
Initial feedback on description

### ISSUE THREAD

**Issue by ldemaz** [Ruphai/ftwfieldmapper#1]
This looks generally well thought out written, with good background information. 

Some clarification/corrections needed:

- On the models you are using, note that the one we use from `fieldmapper`, even though it is named `unet_att_d` with attention capabilities, we do not currently apply that. So this basically a modified U-Net which is deeper and wider than a standard U-Net, and has some dropout layers in it. We are not using DeepLab in this project. Please update this. Also, the model you are comparing against is Field of the World's U-Net model. Provide a brief description of how that differs (this applies to final project, not description, but gives you a head start on write up if you do it now)
- On the various tasks, please:
   - Provide a bit more detail as to tasks entailed in data engineering
   - Note that in evaluating FTW's U-Net you might have to train it from scratch because it currently works on two time points. Or double or time points when you are fine-tuning on it.  
   - The Lacuna labels are not on Source, but on AWS or our cluster

I will provide assessment once this is updated.

**Comment by Ruphai** [comment]
Excellent, thank you @ldemaz, we have now updated those part of the notebook you pointed out. You can find the most recent copy here: https://github.com/Ruphai/ftwfieldmapper/blob/main/notebooks/description.ipynb

**Comment by ldemaz** [comment]
Okay, this looks fine. The description basically gets 20 points for conforming to requirements, and represents the abstract of your project. Note that it will need a bit of updating at the end of the project to summarize key findings and any changes in tasks.  That means in addition to these 20 points, there are 50 more points to earn on the final project (read in detail the requirements on the projects page of the course content). 

Going forward, please provide a new GitHub issue summarizing progress to date, with first issue submitted on Friday this week (end of day), and next issue before our final class Monday.

**Comment by Ruphai** [comment]
Thank you Prof. We will update the issues with the details on our collective accordingly.
